packages/offline-pay-with-crypto-widget/src/components/Modal/index.js

```diff
@@ -36,30 +45,79 @@ const dropIn = {
 
 
 export function Modal({ config }) {
+
+    // eslint-disable-next-line no-unused-vars
+    const { name, order_type, amount, currency, address, store_img } = config;
+
+
   // const [expanded, setExpanded] = React.useState(false);
 
   const [state, setState] = useState("address");
 
-  // eslint-disable-next-line no-unused-vars
-  const { name, order_type, amount, currency, address, store_img } = config;
+  const [selectedCoin, setSelectedCoin] = useState(null)
+
+const paymentInfo = async () => {
+        let goldtoken = await kit.contracts.getGoldToken()
+        const oneGold = kit.web3.utils.toWei('1', 'ether')
+
+    let anAddress = '0x998f2193ae2d887249c27d2ab2ceb9e60be75f7b'
```

Comment:
We do not want to hardcode. New changes fixes this https://github.com/Chimoney/offline-pay-with-celo/blob/main/packages/offline-pay-with-crypto-widget/src/components/Modal/index.js#L39